products/claim-appeal-status/document-status/engineering/drop-offs-after-failures-spike.md

```diff
@@ -0,0 +1,244 @@
+- Currently, exits could reflect two very different things — discouraged users giving up, or users appropriately following offline instructions
+
+### Current State
+
+- **Type 2 Errors & Offline Flow**: Type 2 errors instruct Veterans to resubmit documents offline, which means some exits after these errors are expected and appropriate user behavior.
+
+### Goals
+- Meaningfully differentiate between abandonment and expected offline resubmission behavior
+- Assess whether GA funnel analysis or another approach can reliably distinguish between these cases
+- This work ties to Epic #120479 – Document Status Refinements and Clarifications, specifically supporting the Analytics & Metrics Refinements goal
+
+---
+
+## 4. Technical Findings
+
+### 4.1 Google Analytics Funnel Analysis Capabilities
+
+#### Current Setup
+- **Existing Analytics/Tracking**:
+  - claims-upload-start
+  - claims-upload-cancel
+  - claims-upload-failed
+  - claims-upload-success
+  - page paths (status tab, files tab, needed-from-you, etc.,)
+
+#### Feasibility Assessment
+- GA funnels track can track abandonment drop-offs after errors. What it cannot do is determine why the veteran decided to move on (logout)
+- To get more granularity on the number of documents uploads success/failed (type 1 only), we can update the current recorded events to include that data
+- Type 2 errors do not currently
+
+### 4.3 Platform Analytics Analysis [TBD]
+
+### 4.2 Error Type Analysis
+
+#### Type 1 vs Type 2 Errors
+- **Type 1**: If the file upload fails before it reaches Lighthouse, we can provide immediate feedback to the Veteran that the file was not successfully uploaded. This error is displayed when we are unable to identify what went wrong in the upload attempt. We already provide a variety of specific messages when we know the cause (e.g., duplicate file error). This work doesn't replace existing upload errors.
+  - Fires off the GA event `claims-upload-failure`.
+- **Type 2**: If there is a Type 2 failure, this is typically because the upload has failed between Lighthouse and the eFolder (although not always – see Eng Note). In this case, we will likely not know right away. The Veteran will initially see the "Submission in progress" message (See Successful Submission Flow) and the file will be represented as a card under the "File submissions in progress" section. Once the file fails, which could be up to a few days later, the Veteran will (currently) receive an email telling them their file as failed. For this work, we will add the below alerts to CST to indicate that the failure has occurred. This will trigger a "Files we couldn't receive" section to appear on the page as well.
+  - Does not currently fire off any event (recommendation: add `claims-upload-silent-failure` event with failed document count)
```

Comment:
users do see the alerts but are delayed so should we still labeled it as `silent`?